src/Sylius/Bundle/ApiBundle/Serializer/ProductVariantNormalizer.php

```diff
@@ -67,8 +67,13 @@ public function normalize($object, $format = null, array $context = [])
 
         try {
             $data['price'] = $this->priceCalculator->calculate($object, ['channel' => $this->channelContext->getChannel()]);
+            $originalPrice = $this->priceCalculator->calculateOriginal($object, ['channel' => $this->channelContext->getChannel()]);
+
+            $data['original_price'] = ($data['price'] === $originalPrice ? null : $originalPrice);
```

Comment:
but if `originalPrice` is equal to `price` we should show 2 same value for `price` and `originalPrice` with this code, you will overwrite `originalPrice` :(

what if you will only leave ` data['original_price'] = $originalPrice`